lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1945,16 +1945,14 @@ sub blocked_by_parent_job {
     return undef if $self->is_edge_in_cluster($cluster_jobs);
 
     # now the complicated part
-    my @possibly_blocked_jobs;
```

Comment:
I would have initialized the variable in one line:  
```
my $job_info = $cluster_jobs->{$self->id};
my @possibly_blocked_jobs = ($self->id, @{$job_info->{parallel_parents}}, @{$job_info->{parallel_children}});
```

Same counts for `@own_children`.